great_expectations/execution_engine/sqlalchemy_batch_data.py

```diff
@@ -170,6 +170,8 @@ def __init__(
             else:
                 self._selectable = selectable.alias(self._record_set_name)
 
+        self._engine_ready_selectable = None
```

Comment:
Will you add a docstring to define what this property means?

Reply:
@jcampbell Great idea -- done, and it explains exactly why it is needed and how it works.  Thanks!